lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -555,6 +559,47 @@ class ExtraLinesData {
   }
 }
 
+/// HorizontalRangeAnnotation
+class HorizontalRangeAnnotation {
+  final double y1;
+  final double y2;
+  final Color color;
+
+  HorizontalRangeAnnotation({
+    this.y1,
+    this.y2,
+    this.color = Colors.white,
+  });
+}
+
+/// VerticalRangeAnnotation
+class VerticalRangeAnnotation {
+  final double x1;
+  final double x2;
+  final Color color;
+
+  VerticalRangeAnnotation({
+    this.x1,
+    this.x2,
+    this.color = Colors.white,
+  });
+}
+
+/// RangeAnnotations
+class RangeAnnotations {
+  final bool hasHorizontalRangeAnnotations;
+  final bool hasVerticalRangeAnnotations;
```

Comment:
These fields are redundant because we can check the `horizontalRangeAnnotations` size and if it's empty we shouldn't show anything.
please remove it for the sake of simplicity.

Reply:
24f4dab